src/test/java/io/reactivex/validators/TestPrefixInMethodName.java

```diff
@@ -70,7 +70,7 @@ public void checkAndUpdateTestMethodNames() throws Exception {
                             int lineNum = 0;
                             List<String> lines = new ArrayList<String>();
                             BufferedReader in = new BufferedReader(new FileReader(u));
-                            boolean found = false;
+                            //boolean found = false;
```

Comment:
Remove?

Reply:
It is part of the auto-fix feature currently commented out later in the file. I commented these out so I don't get the unused variable warning. We will enable the auto-fix later in the development.